surfman/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "surfman"
 license = "MIT OR Apache-2.0 OR MPL-2.0"
 edition = "2018"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
I am not sure if we need to bump the minor version here instead, as this patch could break consumers that are working due to the presence of a link to libEGL.so.1 in the binary, similar to servo.